xml/http_cookies.xml

```diff
@@ -5,9 +5,49 @@
   servers.
   -->
 
+  <fingerprint pattern="^__cfd?uid=">
+    <description>CloudFlare web load balancer endpoint</description>
+    <example>__cfuid=1337</example>
+    <example>__cfduid=dd450f2431e1e611a61a15f68974de9a41618794671; expires=Wed, 19-May-21 01:11:11 GMT; path=/; domain=.foo.bar; HttpOnly; SameSite=Lax</example>
```

Comment:
Fingerprint moved up due to frequency of observation.  Coverage for `__cfduid` added